ofl/eduvicwantstarters/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The VIC WA NT School Hand Australia Project Authors (https://github.com/SorkinType/VICWANTSchoolHandAustralia)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+https://scripts.sil.org/OFL
```

Comment:
The url should be https://openfontlicense.org as it is in the font name table (IDs 13 and 14).
Don't know why FB is not reporting this mismatch.

Reply:
@m4rc1e suggested to make a PR to update all the OFLs in one shot. #7482